db/schema.rb

```diff
@@ -632,6 +632,7 @@
     t.datetime "updated_at", precision: nil, null: false
     t.boolean "platform_authenticator"
     t.string "transports", array: true
+    t.jsonb "authenticator_data_flags"
```

Comment:
This doesn't look like it's part of this PR, since it's a change to the webauthn_configurations table.

Reply:
good catch ... i think this somehow snuck in while multi-tasking and switch branches ... thanks 👍🏿